app/routes.js

```diff
@@ -171,6 +171,56 @@ router.get('/form-designer/edit-page/:pageId', function (req, res) {
     // reset the action to avoid a loop
     req.session.data.action = ''
     return res.redirect(`/form-designer/delete/${pageId}`)
+  } else if (action === 'addAnother') {
+    // If user pressed the 'Update preview' button or back link...
```

Comment:
Think this is a copy-paste issue - ideally we should update this comment and the one below `else if (action.includes('removeOption')) {` to describe what these blocks of code are doing, e.g. somrthing like `If user opted to add another choice`

Reply:
Good shout, thanks David. I've gone through and updated the ones I've added, or seen. For the next PR :)